ci/build.py

```diff
@@ -147,30 +176,65 @@ def script_name() -> str:
     docker_binary = get_docker_binary(args.nvidiadocker)
 
     if args.list:
-        platforms = get_platforms()
-        print(platforms)
+        list_platforms()
 
     elif args.platform:
         platform = args.platform
-        if args.build:
+        if args.build or args.build_only:
             build_docker(platform, docker_binary)
+        if args.build_only:
+            return 0
         tag = get_docker_tag(platform)
         if command:
             container_run(platform, docker_binary, command)
+        elif args.docker_run:
+            print(container_run(platform, docker_binary, [""], True))
         else:
             cmd = ["/work/mxnet/ci/docker/runtime_functions.sh", "build_{}".format(platform)]
             logging.info("No command specified, trying default build: %s", ' '.join(cmd))
             container_run(platform, docker_binary, cmd)
 
-    else:
+    elif args.all:
         platforms = get_platforms()
         logging.info("Building for all architectures: {}".format(platforms))
         logging.info("Artifacts will be produced in the build/ directory.")
         for platform in platforms:
             build_docker(platform, docker_binary)
+            if args.build_only:
+                continue
             cmd = ["/work/mxnet/ci/docker/runtime_functions.sh", "build_{}".format(platform)]
-            logging.info("No command specified, trying default build: %s", ' '.join(cmd))
+            shutil.rmtree(buildir(), ignore_errors=True)
             container_run(platform, docker_binary, cmd)
+            plat_buildir = os.path.join(get_mxnet_root(), "build_{}".format(platform))
+            shutil.move(buildir(), plat_buildir)
+            logging.info("Built files left in: %s", plat_buildir)
+
+
+    else:
+        parser.print_help()
+        list_platforms()
+        print("""
+Examples:
+
+./build.py -p armv7
+
+    Will build a docker container with cross compilation tools and build MXNet for armv7 by
+    running: ci/docker/runtime_functions.sh build_armv7 inside the container.
+
+./build.py -p armv7 ls
+
+    Will execute the given command inside the armv7 container
+
+./build.py -p armv7 --docker_run
+
+    Will print a docker run command to get inside the container in an interactive shell
```

Comment:
Would it be possible to just run the command and jump into the container directly rather than requiring the user to copy&paste the command?